prime-dotnet-webapi/Services/Rules/AutomaticAdjudicationRules.cs

```diff
@@ -67,9 +67,19 @@ public override async Task<bool> ProcessRule(Enrollee enrollee)
             foreach (var cert in enrollee.Certifications.Where(c => c.License.Validate))
             {
                 PharmanetCollegeRecord record = null;
+
+                var licenceNumber = cert.License.PrescriberIdType.Equals(PrescriberIdType.NA)
+                                ? cert.LicenseNumber
+                                : cert.PractitionerId;
+
+                if (cert.License.PrescriberIdType.Equals(PrescriberIdType.Optional) && cert.PractitionerId == null)
+                {
+                    return passed;
```

Comment:
I think this should be `continue`, rather than returning.

Reply:
At the same time might as well move this check up to line 69 since its an early exit clause you can check before deciding which number to send to pharmanet